packages/grafana-ui/src/types/icon.ts

```diff
@@ -1,9 +1,23 @@
+import React from 'react';
 import { ComponentSize } from './size';
-export type IconType = 'mono' | 'default';
+
+// Exports of main Icon types
+export type IconProps = SolidIconProps | DefaultIconProps;
+export type IconType = DefaultIconType | SolidIconType;
```

Comment:
`mono` has to be included in the `IconType` as well, since we accept it (at least for now).

Reply:
what is `mono` type?

Reply:
what is `mono` type?